modules/Quiz/resources/quiz.js

```diff
@@ -502,6 +504,19 @@
                     _this.showAnswered(cPo, questionNr);
                 }
             }
+            else if (cPo.submitRequested) {
```

Comment:
long if-else logic. Consider adding a short comment of the case so it will be more readable